clients/roscpp/src/libros/param.cpp

```diff
@@ -472,12 +462,12 @@ bool getCached(const std::string& key, XmlRpc::XmlRpcValue& v)
 	return getImpl(key, v, true);
 }
 
-template <class T> T xml_cast(XmlRpc::XmlRpcValue xml_value) 
+template <class T> T xml_cast(XmlRpc::XmlRpcValue xml_value)
```

Comment:
Please revert this urelated whitespace change.

Same below.

Reply:
Whitespace changes removed.